### PR TITLE
Adapted to pyblep machine description

### PR DESCRIPTION
**Pull request by giadarol**

### CODE REVIEW

File: pysixtrack/track.py [hunk 1]
```diff
@@ -298,16 +296,16 @@ def tobuffer(self):
 
         buffer_list = []
 
-        buffer_list.append(np.array([self.q_part], dtype=np.float64))
-        buffer_list.append(np.array([self.N_part], dtype=np.float64))
+        buffer_list.append(np.array([qe], dtype=np.float64))
+        buffer_list.append(np.array([self.cherge], dtype=np.float64))
```
Comment (rdemaria) [inline pysixtrack/track.py]: typo! charge!

File: pysixtrack/track.py [hunk 2]
```diff
@@ -318,52 +316,75 @@ def tobuffer(self):
...
+
     __units__ = tuple(len(__slots__) * [[]])
-    __defaults__ = tuple(len(__slots__) * [0.])
+    __defaults__ = tuple((len(__slots__)-3) * [0.] + [1e-28, 1e-28, True])
 
     def track(self, p):
         if self.enabled:
             bb6data = BB6Ddata.BB6D_init(
-                self.q_part,
+                qe,
```
Comment (rdemaria) [inline pysixtrack/track.py:355]: where `qe` is defined?

Comment (giadarol) [reply pysixtrack/track.py:355]: It's imported from scipy.constants at the beginning of the file

File: pysixtrack/track.py (hunk 1 above)
Comment (giadarol) [reply pysixtrack/track.py]: This function is unused. I removed it.

**Pull request merged**